servicetalk-grpc-gradle-plugin/src/main/groovy/io/servicetalk/grpc/gradle/plugin/ServiceTalkGrpcPlugin.groovy

```diff
@@ -64,12 +64,11 @@ class ServiceTalkGrpcPlugin implements Plugin<Project> {
         throw new InvalidUserDataException("Please set `serviceTalkGrpc.protobufVersion`.")
       }
 
-      Project rootProject = project.rootProject != null ? project.rootProject : project
       String serviceTalkGrpcProtocGenerateScriptTaskName = "serviceTalkGrpcProtocGenerateScript"
-      if (rootProject.getTasksByName(serviceTalkGrpcProtocGenerateScriptTaskName, false).isEmpty()) {
-        rootProject.task(serviceTalkGrpcProtocGenerateScriptTaskName, {
+      if (project.getTasksByName(serviceTalkGrpcProtocGenerateScriptTaskName, false).isEmpty()) {
```

Comment:
the intention with `rootProject` is that we only need to have a single task which generates the executable script, and have a single script generated in the root project (regardless of how many subprojects apply the plugin). can you clarify why this changed?

Reply:
Understood that intention before making this change. With the `rootProject` approach: 
1. gradle adds `build` folder at the root and does not clean it after with `./gradlew clean`;
2. it increases the exposure of the servicetalk-grpc-protoc plugin for the whole project, that is not necessary if could be isolated in the module that is using it. Because the overhead of this task is very low, it's ok to run it for each module that needs it. Optimization doesn't worth it. Debugging of the gradle's task dependency graph and sequencing of tasks is easier when everything executes as part of a single module.

Reply:
> gradle adds build folder at the root and does not clean it after with ./gradlew clean

This is not the behavior I observe. Can you describe your setup and Is this related to `--include-build` behavior?

Reply:
if we are adding to every sub-project we can get rid of this conditional check and always add the task. which avoids an "expensive" operation for gradle [1].

[1] https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#getTasksByName-java.lang.String-boolean-
> NOTE: This is an expensive operation since it requires all projects to be configured.